rules/docker-base-stretch.mk

```diff
@@ -0,0 +1,17 @@
+# Docker base image (based on Debian Stretch)
+
+DOCKER_BASE_STRETCH = docker-base-stretch.gz
+$(DOCKER_BASE_STRETCH)_PATH = $(DOCKERS_PATH)/docker-base-stretch
+$(DOCKER_BASE_STRETCH)_DEPENDS += $(SUPERVISOR)
+$(DOCKER_BASE_STRETCH)_DEPENDS += $(SOCAT)
```

Comment:
for socat, do we need to compile stretch version of socat?

Reply:
We already are. We are currently building socat v1.7.3.1, which is the current latest version available for Stretch. The version currently available for Jessie is 1.7.2.4.

Reply:
i mean compile the socat in stretch docker.